src/components/App/App.js

```diff
@@ -3,31 +3,17 @@ import { Route, NavLink, Switch } from 'react-router-dom'
 import AllMovies from '../AllMovies/AllMovies'
 import MovieInfo from '../MovieInfo/MovieInfo'
 import Header from '../Header/Header'
-import { fetchAllMovies, fetchMovieId } from '../../utilities/ApiCalls'
+import { fetchAllMovies } from '../../utilities/ApiCalls'
 import './App.css'
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: [],
-      currentMovie: null
+      movies: []
     }
   }
 
-  handleClick = (event) => {
```

Comment:
Love that we moved all this logic out of App into the components where it is actually needed.  Much better than just passing it around as props!  Awesomeness!